mmdet/datasets/xml_style.py

```diff
@@ -123,7 +126,7 @@ def get_ann_info(self, idx):
             ]
             ignore = False
             if self.min_size:
-                assert not self.test_mode
+                # assert not self.test_mode
```

Comment:
is this necessary?

Reply:
I think it's only related to accuracy. So we might uncomment and leave it as before.